test/test_layer_pot.py

```diff
@@ -231,6 +234,19 @@ def test_off_surface_eval_vs_direct(ctx_factory,  do_plot=False):
 
     assert linf_err < 1e-3
 
+    # check that using one FMM works
+    op = op_d.copy(source_kernels=op_d.source_kernels + (knl,),
+        densities=op_d.densities + (sym.var("sigma")*0.5,))
+    single_fmm_bound_op = bind(places, op, auto_where=("fmm_qbx", "target"))
+    print(single_fmm_bound_op.code)
+    single_fmm_fld_in_vol = fmm_bound_op(actx, sigma=fmm_sigma)
+
+    err = actx.np.fabs(fmm_fld_in_vol - single_fmm_fld_in_vol)
+    linf_err = actx.to_numpy(err).max()
+    print("l_inf error:", linf_err)
+
+    assert linf_err < 1e-15
```

Comment:
How come it's sensible to bolt this into an existing test case? (Rather than make it its own?)